framework/db/Schema.php

```diff
@@ -645,4 +594,162 @@ public function isReadQuery($sql)
         $pattern = '/^\s*(SELECT|SHOW|DESCRIBE)\b/i';
         return preg_match($pattern, $sql) > 0;
     }
+
+    /**
+     * Returns the cache key for the specified table name.
+     * @param string $name the table name
+     * @return mixed the cache key
```

Comment:
It's because a key can be [anything](https://github.com/yiisoft/yii2/blob/master/framework/caching/Cache.php#L93) JSON serializable.
I can't change it because it's `protected`, the return type is wide (`mixed`) and we're in 2.0.*